src/Zynga/Framework/Lockable/Cache/V1/Driver/Caching.hh

```diff
@@ -58,37 +58,27 @@ class Caching extends FactoryDriverBase implements DriverInterface {
       throw $e;
     }
   }
-  
-  public function isLocked(StorableObjectInterface $obj): bool {
+
+  public function isLockedByMyThread(StorableObjectInterface $obj): bool {
 
     try {
       $lockKey = $this->getLockCacheKeyFromStorableObject($obj);
       $alreadyLocked = $this->_locks->get($lockKey);
-    
+
       // Check my thead already has a lock
       if ($alreadyLocked instanceof LockPayloadInterface) {
         // Check if the lock is still valid, if so we are done.
-        if ($alreadyLocked->isLockStillValid($this->getConfig()->getLockTTL())) {
+        if ($alreadyLocked->isLockStillValid(
+              $this->getConfig()->getLockTTL(),
+            )) {
           return true;
         }
       }
-      
-      // Check if any other thread has a lock
-      $lockCache = $this->getConfig()->getLockCache();
-      
-      $lockPayload = new LockPayload();
-
-      $lockPayload = $lockCache->get($lockPayload, $lockKey);
-      if($lockPayload === null) {
-        return false;
-      }
-      
-      return $lockPayload->isDefaultValue()[0];
-      
+      return false;
```

Comment:
@Mehul24 You're absolutely correct here, corrected typo. Also took another pass to verify rest of the things., looks good.